CMakeLists.txt

```diff
@@ -755,6 +753,16 @@ if(EXECUTORCH_BUILD_PYBIND)
                  # the torch libs are in `site-packages/torch/lib`.
                  BUILD_RPATH "@loader_path/../../../torch/lib"
                  INSTALL_RPATH "@loader_path/../../../torch/lib"
+                 BUILD_RPATH "@loader_path/../../"
+                 INSTALL_RPATH "@loader_path/../../"
+    )
+  else()
+    set_target_properties(
+      portable_lib
+      PROPERTIES # Assume that this library will be installed in
+                 # `site-packages/executorch/extension/pybindings`, and that
+                 # the custom_ops_aot_lib should be found with relative path.
```

Comment:
Does this mean that custom_ops_aot_lib.so will be installed in the root of the executorch pip package? I'd rather install it in the same directory as portable_lib, or under something like <root>/data/lib.

Reply:
it should be under `site-packages/executorch/extension/llm/custom_ops/`